pytim/gitim.py

```diff
@@ -140,7 +140,7 @@ def __init__(self,
         sanity.assign_alpha(alpha)
 
         self.max_layers = max_layers
-        self._layers = np.empty([max_layers], dtype=type(universe.atoms))
+        self._layers = np.empty([max_layers], dtype=self.universe.atoms[0].__class__)
```

Comment:
Hi,

thanks for the patch! Actually, atoms[0] and atoms are instances of two different classes, I put that back, but you indeed found the problem, which was the missing 'self'. 

This bug comes at the right moment: so far we are basically not testing MDtraj, meaning that many bugs like the one above could be hiding in the code. 

It would be nice to add docstring tests checking MDtraj (basically duplicating some of the ones based on MDAnalysis) but for that, it's better if someone who's actively using MDTraj does the job. Would you like to help us in this direction? Would be much appreciated!

Reply:
Sure. I can write some tests for MDtraj.

Reply:
great, thanks!